src/cozmo/run.py

```diff
@@ -669,7 +671,15 @@ def setup_basic_logging(general_log_level=None, protocol_log_level=None,
             the COMZO_PROTOCOL_LOG_MESSAGES if available which should be
             a comma separated list of message names (case sensitive).
         target (object): The stream to send the log data to; defaults to stderr
```

Comment:
That's always been there, I assume to break up the description from the "defaults to stderr" statement. I think it needs either a semi-colon or a full-stop.